CHANGES/8037.bugfix

```diff
@@ -0,0 +1 @@
+Fix OpenAPI schema pagination
```

Comment:
Any chance you could provide a little more context? I'm not sure I understand the problem that was fixed.